src/transformers/models/auto/tokenization_auto.py

```diff
@@ -28,6 +28,7 @@
 from ..blenderbot.tokenization_blenderbot import BlenderbotTokenizer
 from ..blenderbot_small.tokenization_blenderbot_small import BlenderbotSmallTokenizer
 from ..convbert.tokenization_convbert import ConvBertTokenizer
+from ..cpm.tokenization_cpm import CpmTokenizer
```

Comment:
The CI error is not irrelevant! It's telling you the transformers import crashes because:
```
ModuleNotFoundError: No module named 'sentencepiece'
```

This is because the `CpmTokenizer` is imported here whereas it should be under the 
```
if is_sentencepiece_available():
```

statement.